app/assets/javascripts/discourse/app/initializers/live-development.js

```diff
@@ -30,17 +32,6 @@ export default {
       });
     }
 
-    // Custom header changes
```

Comment:
@CvX couldn't find where this would be used it doesn’t even seem like it would be working today

Reply:
That message-bus event is still (theoretically) being sent by:
https://github.com/discourse/discourse/blob/7f55c9c502d0cafe8ff939bf49f8c573a132e53b/app/models/theme_field.rb#L585-L585

^ that code was added in https://github.com/discourse/discourse/pull/6073 by @OsamaSayegh - so that's the question to Osama: does this works/is it still needed? 😃

Reply:
wait, I didn't read the commit :P Osama just updated that, it was added at the beginning of time by Sam. So… maybe it's no longer needed after all… 🤷

Reply:
If we're dropping it, then the code in theme_field.rb should go too.

Reply:
@OsamaSayegh any opinion on this?

Reply:
Yeah I think we can drop it (and the `publish` call in `theme_field.rb`) -- it seems like in the past we used to wrap themes `header` field HTML with `<header class="custom">...`, but we don't do that anymore.

Reply:
Great, thanks for confirmation

Reply:
Seems like we should remove footer-change too no? I don't even see where it's used but in theme_field.rb